components/rating/rating.html

```diff
@@ -0,0 +1,121 @@
+            <i class="fab fa-github fa-2x"></i>
+        </a>            
+    </div>
+    </header>
+
+    <nav class="navigation">
+        <div class="nav-main-heading">Getting Started</div>
+
+        <div class="nav-sub-heading nav-getstarted">
+            <a href="../home.html">About</a>
+        </div>
+
+        <div class="nav-main-heading">Components</div>
+        <ul class="nav-comp-section">
+            <li class="component-links">
+                <a href="../alert/alert.html">Alert</a>
+            </li>
+            <li class="component-links">
+                <a href="../avatar/avatar.html">Avatar</a>
+            </li>
+            <li class="component-links">
+                <a href="../badge/badge.html">Badge</a>
+            </li>
+            <li class="component-links">
+                <a href="../button/button.html">Button</a>
+            </li>
+            <li class="component-links">
+                <a href="../card/card.html">Card</a>
+            </li>
+            <li class="component-links">
+                <a href="../chips/chips.html">Chips</a>
+            </li>
+            <li class="component-links">
+                <a href="../dialog/dialog.html">Dialog</a>
+            </li>
+            <li class="component-links">
+                <a href="../grid/grid.html">Grid</a>
+            </li>
+            <li class="component-links">
+                <a href="../image/image.html">Image</a>
+            </li>
+            <li class="component-links">
+                <a href="../input/input.html">Input</a>
+            </li>
+            <li class="component-links">
+                <a href="../list/list.html">List</a>
+            </li>
+            <li class="component-links">
+                <a href="../modal/modal.html">Modal</a>
+            </li>
+            <li class="component-links">
+                <a href="../navigation/navigation.html">Navigation</a>
+            </li>
+            <li class="component-links">
+                <a href="../rating/rating.html">Rating</a>
+            </li>
+            <li class="component-links">
+                <a href="../text-utilities/text-utilites.html">Text Utilites</a>
+            </li>
+            <li class="component-links">
+                <a href="../toast/toast.html">Toast</a>
+            </li>
+        </ul>
+    </nav>
+
+    <main class="comp-display">
+        <h1 class="comp-header">Rating</h1>
+        <p class="comp-description">
+            A classification or ranking of someone or something based 
+            on a comparative assessment of their quality, standard, or performance.
+        </p>
+        <h2 class="sub-comp-header">Simple Rating</h2>
+
+        <div class="comp-disp-container">
+            <div class="rating">
+                <span>☆</span>
+                <span>☆</span>
+                <span>☆</span>
+                <span>☆</span>
+                <span>☆</span>
```

Comment:
why not use icons? Why put the stars as text. Just asking, no issues if you want this. Maybe icons would be better i think

Reply:
Suggestion:use font awesome icons its better.

Reply:
I had used icons enough to try something with text. Further, I will use the icons. 
Thanks for the suggestions.